packages/stylelint-plugin/test.less

```diff
@@ -0,0 +1,18 @@
+// This is a file you can use to play with the stylelint plugin
+
+@import "~@kaizen/design-tokens/less/color-vars";
+
+@import "~@kaizen/design-tokens/less/spacing-vars";
+
+.test {
+  color: @kz-var-color-wisteria-800;
+}
+
+.testt {
```

Comment:
This looks like a typo, but I'm guessing it's to differentiate from the other class?

Reply:
yep just a play file, cleaned it up